service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -67,7 +67,46 @@ void setUp() {
 
     @Test
     void shouldReturnHearingDateDynamicListWhenCaseDetailsHasHearingDate() {
-        hearingDetails = createHearingBookings(NOW);
+        hearingDetails = createHearingBookings(NOW.plusDays(5));
+        caseManagementOrder = CaseManagementOrder.builder().build();
+
+        Map<String, Object> data = draftCMOService.extractIndividualCaseManagementOrderObjects(
+            caseManagementOrder, hearingDetails);
+
+        DynamicList hearingList = (DynamicList) data.get("cmoHearingDateList");
+
+        assertThat(hearingList.getListItems())
+            .containsAll(Arrays.asList(
+                DynamicListElement.builder()
+                    .code(fromString("b15eb00f-e151-47f2-8e5f-374cc6fc2657"))
+                    .label(formatLocalDateToMediumStyle(10))
```

Comment:
The method name is not clear enough what 10 means. I know it's not your code but maybe we would benefit from super quick refactor?

Reply:
fine to do a little refactor. Just wanted to get this done ASAP. Although I feel the bigger problem here is the obscuring of the test data by using the horrible CaseDataGeneration stuff.

Reply:
ok, having started, I will not proceed. Let's do tech debt.

Reply:
https://tools.hmcts.net/jira/browse/FPLA-1108

Reply:
Thanks Sam.